dev/mongo-rs-backups/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM percona/percona-server-mongodb:4.2
```

Comment:
Maybe we can make the image tag configurable? Not super important, but e.g 4.4 is also out (and they are other versions)

Reply:
Probably, but for now let's leave it as is. It's just starting point to simplify dev environment creation, we will improve it over time.